Microsoft.Toolkit.Uwp.UI.Controls/MosaicControl/MosaicControl.cs

```diff
@@ -0,0 +1,1112 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Numerics;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Robmikh.CompositionSurfaceFactory;
+    using Windows.ApplicationModel;
+    using Windows.Foundation;
+    using Windows.Foundation.Metadata;
+    using Windows.Storage;
+    using Windows.UI.Composition;
+    using Windows.UI.Xaml;
+    using Windows.UI.Xaml.Controls;
+    using Windows.UI.Xaml.Hosting;
+    using Windows.UI.Xaml.Media;
+    using Windows.UI.Xaml.Media.Imaging;
+    using Windows.UI.Xaml.Shapes;
+
+    public enum UIStrategy
+    {
+        /// <summary>
+        /// MosaicControl is created with XAML
+        /// </summary>
+        PureXaml,
```

Comment:
Why exposing this? I understand the need to use Xaml when composition is not available (and I love this fallback) but why as an user could I change it?